src/YearZeroRoll.js

```diff
@@ -572,6 +572,12 @@ export default class YearZeroRoll extends Roll {
       else this._total = 0;
     }
 
+    const terms = this.terms;
+    // eslint-disable-next-line no-undef
+    if (terms[0] instanceof OperatorTerm) {
+      terms.shift();
+    }
```

Comment:
I propose to move this block above the line 568 where the formula is updated.

Reply:
done and done